helm/chaos-mesh/templates/chaos-daemon-rbac.yaml

```diff
@@ -88,15 +88,15 @@ spec:
     - IPC_LOCK
   {{- end }}
   allowedHostPaths:
-    - pathPrefix: {{ .Values.chaosDaemon.socketPath }}
+    - pathPrefix: {{ .Values.chaosDaemon.socketPath | dir }}
```

Comment:
it would be better if we set `allowedHostPaths ` according to this block of code:

https://github.com/chaos-mesh/chaos-mesh/blob/master/helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml#L187-L200

Reply:
Hi @stephen-harris , please take a look :heart:

Reply:
@xlgao-zju I've moved the logic to a helper and used it in both the daemonset and psp